src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/SearchService.kt

```diff
@@ -38,6 +38,8 @@ class SearchService(
     return matches.firstOrNull()?.candidateRecord
   }
 
+  fun retrieveRecordSelfMatchScore(person: Person): Double = matchService.getSelfMatchScore(person)
```

Comment:
It is here as well as we will need to apply a threshold filter to it in the future